### PR TITLE
changes to support FEATURE_DNS_QUERY

### PR DESCRIPTION
**Pull request by bill1600**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -82,7 +83,9 @@ add_dependencies(libtrower-base64 trower-base64)
 ExternalProject_Add(nopoll
     PREFIX ${PREFIX_DIR}/nopoll
     GIT_REPOSITORY https://github.com/Comcast/nopoll.git
-    GIT_TAG "nopoll_yocto"
+    #GIT_TAG "8033547de94c2da1e1948e12e90a5b305af477c2"
+    #GIT_TAG "nopoll_yocto"
+	GIT_TAG "18e83e32bfccd1f308b4df0fe0aa3565302903a4"
```
Comment (shilpa24balaji) [inline CMakeLists.txt]: Is it necessary to go back in nopoll version, any issue with latest?

File: CMakeLists.txt [hunk 2]
```diff
@@ -82,7 +83,9 @@ add_dependencies(libtrower-base64 trower-base64)
 ExternalProject_Add(nopoll
     PREFIX ${PREFIX_DIR}/nopoll
     GIT_REPOSITORY https://github.com/Comcast/nopoll.git
-    GIT_TAG "nopoll_yocto"
+    #GIT_TAG "8033547de94c2da1e1948e12e90a5b305af477c2"
```
Comment (shilpa24balaji) [inline CMakeLists.txt]: Please remove commented lines

**Comment by bill1600** [comment]
Merged directly

**Comment by bill1600** [comment]
Merged directly

**Pull request closed without merging**